cubby-app/src/cmr/cubby/system.clj

```diff
@@ -46,7 +46,7 @@
              :scheduler (jobs/create-scheduler
                          `system-holder
                          [jvm-info/log-jvm-statistics-job])}]
-    (transmit-config/system-with-connections sys [:echo-rest])))
+    (transmit-config/system-with-connections sys [:access-control :echo-rest])))
```

Comment:
We still get ACLs from echo-rest.  I trie to change that (even though it is out of scope) when I thought it would make tests simpler, but it broke things even worse, so we will need to handle that separately